### ISSUE TITLE
What database do you use with plots2? (poll)

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#4919]
Hi all, we're curious what database do you use with plots2? We're discussing in #4896 whether to continue supporting sqlite or not -- and want to know how many people currently use different database types!

/polls MySQL MariaDB SQLite Other

Re: #4917

**Comment by namangupta01** [comment]
On local-machine or in development,  I prefer to use SQLite because it is very easy to set up, drop etc. So my vote goes to SQLite.

**Comment by sashadev-sky** [comment]
@namangupta01 do you have mysql installed on your system as well and just choose to use sqlite? I was wondering if you'd be able to to try and see if you receive this error in plots2: https://github.com/publiclab/plots2/issues/2712#issuecomment-468382671

**Comment by namangupta01** [comment]
Yes, @sashadev-sky, I do have MySQL installed on my computer and even then I try to use SQLite wherever I could.
Let me check for the error. I have run the test using SQLite several times but I didn't get any such type of error. But I think I should try to test it now to check if this problem is with everyone else.

**Comment by jywarren** [comment]
thank you naman and sasha!

**Comment by siaw23** [comment]
@namangupta01 I'm interested in knowing what you get when you ran `rake db:reset` and `rake db:setup` while on SQLite. I'd be happy to know if nothing breaks for you! Mind sharing this info?

**Comment by jywarren** [comment]
And @namangupta01 -- if we did choose to deprecate sqlite support, do you think it would be very disruptive to you? We're trying to weigh the pros and cons, but leaning towards using just one DB. Thank you for your input!

**Comment by sashadev-sky** [comment]
@namangupta01 thank you!! @siaw23 you have a much more in depth understanding of the sqlite issue so i'll leave it to you to investigate with @namangupta01 but i'm also interested so keep me in the loop if you open any new issues on it, etc!

**Comment by gauravano** [comment]
Hi everyone, let's move forward with this unless we want to have more contributors' opinion on this one. Also, if we decide on closing polls, do copy the result in #4896 and let's continue the discussion there only. 

Thanks!
